### ISSUE TITLE
How to get saliency maps in fig6?

### ISSUE THREAD

**Issue by Magallan1229** [MadryLab/backgrounds_challenge#2]

**Comment by kaixiao** [comment]
Hi,

We compute the gradient of the image with respect to the loss to get the saliency map.

We then use the following code to visualize the gradient:

```
def vis_grad(grad):
    std = 3 * torch.std(grad)
    img = torch.clamp(grad, -std, std)
    img = (1 + grad / std) * 0.5
    return torch.tensor(grad)

```

**Comment by Magallan1229** [comment]
Thank you for your reply!
